previews/primer/beta/flash_preview.rb

```diff
@@ -4,16 +4,28 @@ module Primer
   module Beta
     # @label Flash
     class FlashPreview < ViewComponent::Preview
-      # @label Default
+      # @label Playground
+      #
+      # @param full toggle
+      # @param full_when_narrow toggle
+      # @param dismissible toggle
+      # @param icon [Symbol] select [none, alert, check, info, people]
+      # @param scheme [Symbol] select [default, warning, danger, success]
+      # @param content text
+      # @param description text
+      def playground(full: false, full_when_narrow: false, dismissible: false, icon: :people, scheme: Primer::Beta::Flash::DEFAULT_SCHEME, content: "This is a flash message!", description: nil)
+        render(Primer::Beta::Flash.new(full: full, full_when_narrow: full_when_narrow, dismissible: dismissible, icon: icon == :none ? nil : icon, scheme: scheme, description: description)) { content }
+      end
+
+      # @label With action button
       #
       # @param full toggle
-      # @param spacious toggle
       # @param dismissible toggle
       # @param icon [Symbol] select [alert, check, info, people]
       # @param scheme [Symbol] select [default, warning, danger, success]
       # @param content text
-      def default(full: false, spacious: false, dismissible: false, icon: :people, scheme: Primer::Beta::Flash::DEFAULT_SCHEME, content: "This is a flash message!")
```

Comment:
We'll need to keep at least one `default` preview. see https://github.com/github/primer/pull/1387